### ISSUE TITLE
Keras API create graph model would get error

### ISSUE THREAD

**Issue by jenniew** [intel-analytics/BigDL#2876]
Using Keras API to create graph mode get this error: 
com.intel.analytics.bigdl.nn.abstractnn.InvalidLayer: Do not mix StaticGraph[15c23a21](isKerasStyle=false) with Layer
                           (isKerasStyle=true):
         Input[f7b728aa],Bidirectional[b2228acb],Bidirectional[2e54b783],Dense[243b03cf]

Example code: 
        from bigdl.nn.keras.topology import *
        from bigdl.nn.keras.layer import Input, Bidirectional, LSTM, Dense
        from bigdl.optim.optimizer import *

        input1 = Input(shape=(options.max_seq_length, 768))
        lstm1 = Bidirectional(LSTM(768, return_sequences=True))(input1)
        lstm2 = Bidirectional(LSTM(768, return_sequences=True))(lstm1)
        fc = Dense(len(label_list), activation="softmax")(lstm2)
        model = Model(inputs=input1, outputs=fc)

**Comment by jason-dai** [comment]
from zoo.pipeline.api.keras.layers import *?

**Comment by hkvision** [comment]
`model = Model(inputs=input1, outputs=fc)` In BigDL, it should be `input/output` rather than `inputs/outputs`?
I tried the above code and there should be no problem.
You are recommended to use zoo keras layers.

**Comment by hkvision** [comment]
Seems you are using the `Model` under `bigdl.nn.layer` but not `bigdl.nn.keras.layer`

**Comment by jenniew** [comment]
Even change to input/output, still get this error. No, I use bigdl.nn.keras.layer, not bigdl.nn.layer.  The error is triggered by optimizier.optimize()

**Comment by jason-dai** [comment]
Please use `zoo.pipeline.api.keras.layers`

**Comment by jenniew** [comment]
Changed to zoo keras api, still get this error. And the issue only occurs in  mkldnn

**Comment by hkvision** [comment]
Seems it's the same issue with https://github.com/intel-analytics/analytics-zoo/issues/1534
